apps/frontend/src/config.ts

```diff
@@ -4,7 +4,7 @@ export const API_BASE_URL =
 export const ALGOLIA_APP_ID =
   process.env.REACT_APP_ALGOLIA_APP_ID || 'LVYWOPQ0A9';
 export const ALGOLIA_INDEX =
-  process.env.REACT_APP_ALGOLIA_INDEX || 'asap-hub_research_outputs_dev';
+  process.env.REACT_APP_ALGOLIA_INDEX || 'asap-hub_dev';
```

Comment:
This should be behind feature flag

Reply:
The default value is only for local development - so this should be fine, no?